docs/components/navigator.md

```diff
@@ -28,7 +28,7 @@ var navigatorInstanceOne = gamepad.navigator();
 // Otherwise create a custom component using the navigator grade.
 fluid.defaults("my.navigator.grade", {
     gradeNames: ["gamepad.navigator"]
-};
+});
```

Comment:
I totally missed that.  This is the kind of thing that linting checks are really good at.